OpenSim/Common/AbstractProperty.h

```diff
@@ -104,6 +104,14 @@ class OSIMCOMMON_API AbstractProperty {
     // TODO: replace this with something more reasonable
     virtual std::string toString() const = 0;
 
+    /** For relatively simple types, return the current value of this property
+    in a string suitable for displaying to a user in the GUI. Objects just
+    return something like "(Object)". This differs from `toString()` as it has
+    an optional argument, `precision`, for controlling the number of digits
+    printed to string. **/
```

Comment:
Consider adding that if this function is not overridden in a derived class (e.g., deprecated properties), this function uses `toString()` and the precision argument is ignored.

Reply:
I just read about how default arguments work with virtual functions (https://stackoverflow.com/questions/3533589/can-virtual-functions-have-default-parameters). Seems to be a bit of a mess. I'm wondering if it would be best to put the default argument in the base class, and to not provide a default argument in any of the derived classes, because I think people are more likely to call the method from the base class than the derived class.